llvm/lib/Target/LoongArch/LoongArchMCInstLower.cpp

```diff
@@ -35,10 +35,10 @@ static MCOperand lowerSymbolOperand(const MachineOperand &MO, MCSymbol *Sym,
     Kind = LoongArchMCExpr::VK_None;
     break;
   case LoongArchII::MO_CALL:
-    Kind = LoongArchMCExpr::VK_CALL;
+    Kind = ELF::R_LARCH_B26;
     break;
   case LoongArchII::MO_CALL_PLT:
-    Kind = LoongArchMCExpr::VK_CALL_PLT;
+    Kind = ELF::R_LARCH_B26;
```

Comment:
Seems these 2 cases can be merged into one.

Reply:
Done. The two MO_ can probably be merged, but that should be a future change.